editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper.spec.browser.tsx

```diff
@@ -161,9 +161,7 @@ describe('Spy Wrapper Template Path Tests', () => {
     expect(sanitizedDomMetadata).toMatchInlineSnapshot(`
       Object {
         ":storyboard": Object {
-          "children": Array [
-            ":storyboard/scene",
-          ],
+          "children": Array [],
```

Comment:
@Rheeseyb heads up, I think this change is "correct" and nothing depends on this behavior (this is how it was 5 days ago).

The reason we don't see children here because the dom-walker now treats scenes as scenes again, and "restarts" the walk when it encounters one.